README.md

```diff
@@ -56,6 +50,31 @@ During local development API requests are routed through a [webpack-dev-server p
 
 If you have access to the trusted custom CA certificates you can add them to Node.js by setting up your environment as follows:
 
-```
+```bash
 export NODE_EXTRA_CA_CERTS=/path/to/your/cert.pem
 ```
+
+### I do not have permission to add `http://localhost:8080` to my CORS policy in control panel
+
+If you do not have permission to do so and can't get Foundry Admin to approve it you can use the following setup:
+
+1. Add `https://localhost:8080/auth/callback` as a redirect URL in the Permission & OAuth tab on the left menu.
+1. Run `mv .env.development.sample .env.development` to start setting up your environment variables, and fill in the fields marked with `<>`
+1. Add the following section to your `devServer: ` section in the webpack.config.js file, after `port: 8080,`
+
+```javaScript
+    server: useLocalhostInCORS ? "http" : "https",
+    proxy: useLocalhostInCORS
+        ? []
+        : [
+                {
+                    // This proxies calls from the browser to the configured Foundry instance
+                    target: process.env.API_PROXY_TARGET_URL,
+                    context: ["/multipass/api/**", "/api/**"],
+                    changeOrigin: true,
+                    secure: true,
+                },
+            ],
+```
+
+4. Test your application by running `npm run dev` and visiting https://localhost:8080 (note HTTPS and not HTTP)
```

Comment:
might be worth leaving a note saying that you'll get a this is unsafe error, and that is expected

Reply:
Done!